Scenes/Server/Server.gd

```diff
@@ -29,6 +29,5 @@ func _ready() -> void:
 	for member in members:
 		var member_button = member_button_scene.instance()
 		member_button.name = member["user"]["username"]
-		member_button.text = member["user"]["username"]
```

Comment:
This is the line I am talking about! No need, see `member_button.gd`, it is already there!